packages/config/src/layer2s/starknet.ts

```diff
@@ -429,7 +429,7 @@ export const starknet: Layer2 = {
             '2479841346739966073527450029179698923866252973805981504232089731754042431018',
           ],
         },
-        _hackCostMultiplier: 0.85,
+        _hackCostMultiplier: 0.65,
```

Comment:
Shouldn't it be 0.7 here? Since the untilTimestamp is 12.03?

Reply:
[ca2413b](https://github.com/l2beat/l2beat/pull/3371/commits/ca2413b3f51f028c0208b72dcbe31c14a0225f44)